lib/vyasa_web/components/contexts/read.ex

```diff
@@ -226,21 +226,10 @@ defmodule VyasaWeb.Context.Read do
             }
           }
         } = socket
-      ) do
-    reply_to_sheaf =
-      case parent_id do
-        p_id when is_binary(p_id) ->
-          Sangh.get_sheaf(p_id)
-
-        p when is_nil(p) ->
-          nil
-
-        _ ->
-          nil
-      end
```

Comment:
this is the previous PR's stuff that I hadn't added in before merging the previous, approved PR.